.gitmodules

```diff
@@ -1,3 +1,8 @@
+[submodule "lib/forge-std"]
```

Comment:
Not sure if this is doable or viable but optimally we could also use `forge-std` which is already imported in `lib/espresso-tee-contracts` to avoid redundancy. When I tried in another branch it was able to build and run successfully.

Reply:
I am worried that arbitrum labs would want a different version of forge std than what we will have in lib/espresso-tee-contracts ( probably they will support an older version)